services/brig/test/integration/API/User/Client.hs

```diff
@@ -51,10 +51,10 @@ tests _cl _at _conf p b c g = testGroup "client"
 
 testAddGetClient :: Brig -> Cannon -> Http ()
 testAddGetClient brig cannon = do
-    u <- randomUser brig
```

Comment:
This clutter is here because I changed `addClient` to not require a full `User`. This is the right thing to do, but it does make the diff messier :pensive:

Reply:
In fact, this whole module can be skipped from review -- it contains nothing but this diff clutter.